packages/contracts/src/dollar/utils/CollectableDust.sol

```diff
@@ -24,7 +24,7 @@ abstract contract CollectableDust is ICollectableDust {
             !_protocolTokens.contains(_token),
             "collectable-dust::token-is-part-of-the-protocol"
         );
-        _protocolTokens.add(_token);
+        require(_protocolTokens.add(_token));
```

Comment:
Curious about this added `require`

Reply:
Both the EnumerableSet and StructurededLinkList will return a bool for functions that alter the sets. Slither was calling these out as being ignored and a require was the simplest way to ensure a revert on failure.